package.json

```diff
@@ -23,17 +23,21 @@
   "license": "UNLICENSED",
   "peerDependencies": {
     "lodash": "^4.17.2",
-    "react": "~15.5.0",
+    "react": "^15.6.1",
     "react-dom": "~15.5.0"
```

Comment:
Should `styled-components` also be a peer dependency, since service-ui uses it now?